content/api-references/accounts/accounts.md

```diff
@@ -557,6 +546,233 @@ _Some server error occurred. Please contact Alpaca._
+| `data_comparison`       | enum.CIPResult    | Checks whether data on the document is consistent with data provided when creating an applicant through the API |
+| `data_comparison_breakdown`          | object    | example: {"date_of_birth": "clear", "date_of_expiry": "clear" "document_numbers": "clear", "document_type": "clear", "first_name": "clear", "gender": "clear", "issuing_country": "clear", "last_name": "clear"} |
+| `image_integrity`      | enum.CIPResult    | Checks whether the document was of sufficient quality to verify |
+| `image_integrity_breakdown`          | object | example: {"colour_picture": "clear", "conclusive_document_quality": "clear", "image_quality": "clear", "supported_document": "clear"} |
+| `visual_authenticity`          | object | Checks whether visual (non-textual) elements are correct given the document type. Example: {"digital_tampering": "clear", "face_detection": "clear", "fonts": "clear", "original_document_present": "clear", "picture_face_integrity": "clear", "security_features": "clear", "template": "clear"} |
+
+
+**Photo**
+
+| Attribute               | Type        | Notes                              | 
+| ----------------------- | ----------- | ---------------------------------- | 
+| `id`                    | string/UUID | Your internal ID of check          |
+| `result`                | enum.CIPResult         | Overall result of specific check   |
+| `status`                | enum.CIPStatus      | Overall status of specific check   |
+| `created_at`            | timestamp   |  |
+| `face_comparison`       | enum.CIPResult        | Checks whether the face in the document matches the face in the live photo  | 
+| `face_comparison_breakdown`        | object        | {"face_match":{"result": "clear","properties":{"score": "80"}}} | 
+| `image_integrity`       | enum.CIPResult        | Checks whether the quality and integrity of the uploaded files were sufficient to perform a face comparison |
+| `image_integrity_breakdown`         | string      | example: {"face_detected":{"result": "clear"},"source_integrity": {"result": "clear"}} |
+| `visual_authenticity`   | enum.CIPResult      | Checks whether the person in the live photo is real (not a spoof) |
+| `visual_authenticity_breakdown`             | string      | {"spoofing_detection": {"result": "clear","properties": {"score": "26"}}}} |
+
+
+**Identity**
+
+| Attribute               | Type        | Notes                              | 
+| ----------------------- | ----------- | ---------------------------------- | 
+| `id`                    | string/UUID | Your internal ID of check          |
+| `result`                | enum.CIPResult         | Overall result of specific check   |
+| `status`                | enum.CIPStatus      | Overall status of specific check   |
+| `created_at`            | timestamp   |  |
+| `matched_address`       | enum.CIPResult   | Represents the matched address for the applicant |
```

Comment:
This is a string, which I believe should match an ID in the matched_addresses below